documentation/modules/configuring/con-config-mirrormaker2-replication.adoc

```diff
@@ -81,6 +85,28 @@ Both topics enable replication to be fully restored from the correct offset posi
 
 MirrorMaker 2.0 uses its `MirrorCheckpointConnector` to emit _checkpoints_ for offset tracking.
 
+== Synchronizing consumer group offsets
+
+The `__consumer_offsets` topic stores information on committed offsets, according to consumer group.
+Offset synchronization periodically transfers the consumer offsets for the consumer groups of a source cluster into the consumer offsets topic of a target cluster.
+
+Offset synchronization is particularly useful in an _active/passive_ configuration.
+Should the active cluster go down, consumer applications can switch to the passive (standby) cluster and pick up from the last offset position.
+
+To use topic offset synchronization:
+
+* Enable the synchronization by adding `sync.group.offsets.enabled` to the checkpoint connector configuration of the `KafkaMirrorMaker2` resource,
+and setting the property to `true`. Synchronization is disabled by default.
+* Add the `IdentityReplicationPolicy` to the source connector configuration so that topics in the target cluster retain their original names.
+
+For topic offset synchronization to work, consumer groups in the source cluster cannot be active in the target cluster.
+Consequently, if you are using an _active/active_ configuration that does not support or require offset synchronization, do not enable topic offset synchronization.
```

Comment:
I'm not sure I understand this sentence. In general, you can use offset synchronization also in active-active clusters. And since you anyway cannot easily use a single consumer group over multiple clusters, this might not be a big limitation I think. So maybe we can remove this sentence?

Reply:
Removed